src/clib/pioc_support.c

```diff
@@ -3375,11 +3375,38 @@ int pioc_change_def(int ncid, int is_enddef)
     {
         LOG((3, "pioc_change_def calling netcdf function file->fh = %d file->do_io = %d iotype = %d",
              file->fh, file->do_io, file->iotype));
+
+        /* NetCDF and PnetCDF by default do not reserve any extra space in
+         * the NetCDF file headers (compactness etc). However this can be a
+         * problem (performance related) when renaming variables etc since
+         * this information is stored in the NetCDF header. Adding more
+         * information into the header requires the libraries to resize the
+         * file, extend the header space and copy the contents of the file
+         * (similar to realloc).
+         *
+         * The solution for the performance issue is to reserve some extra
+         * space in the header when creating NetCDF files. The current calls
+         * to nc_enddef()/ncmpi_enddef() need to be replaced with nc__enddef
+         * ()/ncmpi__enddef() (note the double underscore).
```

Comment:
Can you reword the commit message? Especially the part,
```
The solution for the performance issue is to reserve some extra
space in the header when creating NetCDF files. The current calls
to nc_enddef()/ncmpi_enddef() need to be replaced with nc__enddef
()/ncmpi__enddef() (note the double underscore).
```
can be changed to something like
```
So reserving some extra space in the header when creating
NetCDF files by replacing current calls to
nc_enddef()/ncmpi_enddef() with
nc__enddef()/ncmpi__enddef() that allow reserving extra
space in the header
```

(Note: The comment here does not need to change)

Reply:
OK, I will update it.